recipes/scikit-image/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "0.12.3" %}
+
+package:
+  name: scikit-image
+  version: {{ version }}
+
+source:
+  fn: scikit-image-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/scikit-image/scikit-image-{{ version }}.tar.gz
+  md5: 04ea833383e0b6ad5f65da21292c25e1
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - setuptools
+    - cython
+    - python
+    - six
+    - numpy x.x
```

Comment:
Yeah. So, this is a particular oddity in `conda-build`. That specifies we are linking to `numpy` and we expect to match the same major and minor version from build-time at run-time. There has been discussion about expanding this to other packages, as well.